db/migrate/20200701133311_create_review_turnarounds.rb

```diff
@@ -3,6 +3,8 @@ def change
     create_table :review_turnarounds do |t|
       t.references :review_request, null: false, foreign_key: true
       t.integer :value
+
+      t.timestamps
```

Comment:
you shouldn't be editing existing migrations (if they were already run in Heroku), you need to create new migrations

Reply:
Is not in heroku yet